### PR TITLE
Extend API of TextField

### PR DESCRIPTION
**Pull request by Dimon4eg**
Added extended methods to TextField:
- getStringNumLines
- getLineHeight
- setLineBreakWithoutSpace

### CODE REVIEW

File: cocos/ui/UITextField.h [hunk 1]
```diff
@@ -622,7 +622,26 @@ class CC_GUI_DLL TextField : public Widget
...
+
+    /**
+     * Return the number of lines of text.
+     */
+    int getStringNumLines();
+
+    /**
+     * Returns the line height.
+     * @warning Not support system font.
+     * @since v3.2.0
```
Comment (zilongshanren) [inline cocos/ui/UITextField.h]: Why `@since v3.2.0`?  It should be `@since v3.9`?

Comment (Dimon4eg) [inline cocos/ui/UITextField.h]: Yes, you're right. It's just copy-paste from Label.

**Comment by Dimon4eg** [comment]
I've merged to v3.8.1 branch. Is this ok?

**Comment by zilongshanren** [comment]
@Dimon4eg 
you'd better resend a PR to v3 branch.
Thanks.

**Pull request closed without merging**